api/lib/gateways/Jigsaw/JigsawSearchGateway.js

```diff
@@ -0,0 +1,51 @@
+const { dedupe } = require('../../Utils');
+const { Systems } = require('../../Constants');
+
+module.exports = options => {
+  const buildSearchRecord = options.buildSearchRecord;
+  const doJigsawGetRequest = options.doJigsawGetRequest;
+  const jigsawEnv = options.jigsawEnv;
+
+  const searchUrl = `https://zebracustomers${jigsawEnv}.azurewebsites.net/api/customerSearch`;
+
+  const search = async query => {
+    const search = [query.firstName, query.lastName].filter(x => x).join(' ');
+    return await doJigsawGetRequest(searchUrl, { search });
+  };
+
+  const validateIds = record => {
+    return record.id;
+  };
+
+  const processRecords = records => {
+    if (records.length == 1 && records[0].id == 0) {
```

Comment:
Please use === here when testing equality (see https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons)